lib/hypervisor/hv_kvm/monitor.py

```diff
@@ -732,7 +692,7 @@ def HasDynamicAutoReadOnly(self):
       return True
 
     return any([x for x in schema
-                if "dynamic-auto-read-only" in x.get("features",[])])
+                if "dynamic-auto-read-only" in x.get("features", [])])
```

Comment:
extra whitespace

Reply:
pycharm has reported this

Reply:
PEP 8: E231 missing whitespace after ','
https://www.flake8rules.com/rules/E231.html

Reply:
Also acceptable argument. I should definitely create an issue to not forget writing style guidelines ...